lab-davide/route/note-router.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+//node modules
+//npm modules
+//app modules
+//global
+
+//we are requiring the Router function from the express module//
+const Router = require('express').Router;
+const noteRouter = module.exports = new Router();
+const debug = require('debug')('note:note-router');
+const JsonParser = require('body-parser').json();
+const storage = require('./../lib/storage');
+const Note = require('./../model/note');
+
+
+function createNote(reqBody) {
+  debug('createNote');
+  return new Promise(function(resolve, reject) {
+    var note;
+    try {
+      note = new Note(reqBody.content);
+    } catch (err) {
+      reject(err);
+    }
+    storage.setItem('note', note).then(function(note) {
+      resolve(note);
+    }).catch(function(err) {
+      reject(err);
+    });
+  });
+}
+
+//adding POST request//
+noteRouter.post('/', JsonParser, function(req, res) {
```

Comment:
It might improve readability if you change your formatting to something like this: 

```javascript
createNote(req.body)
.then(function(note) {
  res.status(200).json(note);
})
.catch(function(err) {
  console.error(err.message);
})
```